doc/cookbook/source/examples/classifier/LDA.rst

```diff
@@ -0,0 +1,52 @@
+=============================
+Linear Discriminant Analysis
+=============================
+
+LDA learns a linear classifier that attempts to find a projection matrix that maximally discriminates the provided classes. The learned linear classification rule is optimal under the assumption that both classes a gaussian distributed with equal co-variance. To find a linear separation :math:`{\bf w}` in training, the in-between class variance is maximized and the within class variance is minimized.
+Methods that can be used to compute the LDA are:
+
+Two class Fisher Discriminant Analysis:
+
+.. math::
+
+	J({\bf w})=\frac{{\bf w^T} S_B {\bf w}}{{\bf w^T} S_W {\bf w}}
+
+Singular Valued decomposition method:  The above derivation of Fisher's LDA requires the invertibility of the within class matrix. However, this condition gets void when there are fewer data-points than dimensions. In this case SVD is used.
```

Comment:
This condition, however, is violated when there ....

(Never start a sentence with however)